src/platform/windows/events_loop.rs

```diff
@@ -281,7 +281,7 @@ pub unsafe extern "system" fn callback(window: winapi::HWND, msg: winapi::UINT,
                                        -> winapi::LRESULT
 {
     match msg {
-        winapi::WM_DESTROY => {
+        winapi::WM_CLOSE => {
             use events::WindowEvent::Closed;
             send_event(Event::WindowEvent {
```

Comment:
Window removal might be postponed to WM_DESTROY, and only the event es emitted here, but i'm not sure if it is required.